components/financial_assistance/app/models/financial_assistance/applicant.rb

```diff
@@ -178,6 +178,10 @@ class Applicant # rubocop:disable Metrics/ClassLength TODO: Remove this
     field :family_member_id, type: BSON::ObjectId
     field :eligibility_determination_id, type: BSON::ObjectId
 
+    # @deprecated This field is implemented to soft delete the applicant from the system just like the family member.
```

Comment:
Small note-- may be more clear to say "This field `was` implemented to soft delete the applicant from the system just like the family member.  However, the applicant can `now` be deleted from the system as it is only allowed for draft applications, so we are not using this field anymore."

Reply:
I will update the comment in the next PR.